test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/base/PipelineBaseE2EIT.java

```diff
@@ -109,6 +109,7 @@ public abstract class PipelineBaseE2EIT {
     private Thread increaseTaskThread;
     
     public PipelineBaseE2EIT(final PipelineTestParameter testParam) {
+        log.info("test E2EIT class : {}, parameterized:{}", this.getClass().getName(), testParam);
```

Comment:
1, It could be removed

2, Could we add log at real unit test method entrance, so we could find it in CI for every real test when it start